lib/iris/fileformats/grib/__init__.py

```diff
@@ -926,9 +926,61 @@ def load_cubes(filenames, callback=None, auto_regularise=True):
     return iris.fileformats.rules.load_cubes(filenames, callback, grib_loader)
 
 
+def load_pairs_from_fields(grib_messages):
+    """
+    Convert an iterable of GRIB messages into an iterable of
+    (Cube, Grib message) tuples.
+
+    Args:
+
+    * grib_messages:
+        An iterable of :class:`iris.fileformats.grib._message._GribMessage`.
```

Comment:
Since we now made 'GribMessage' public, this reference is now out-of-date, i.e. want to remove the underscores